npc/quests/quests_13_2.txt

```diff
@@ -4481,6 +4482,11 @@ function	script	jewel_13_2	{
 		close;
 	}
 	else if (ep13_2_rhea == 5) {
+		getinventorylist;
+		if ( @inventorylist_count > 98 ) { //Need this check to solve/prevent the bug that failed to get the item.
```

Comment:
I don't think it's nice to set this check to 98, because many servers increase inventory limit eg 150 ~ 200... And another example is how I use **Inventory Expanded** the default limit goes up to 200

Reply:
Okay so, what about script to check max inventory limit then -2 (maxlimit-2)?

Reply:
Uses something like `if (!checkweight(1201,2)) {` instead, the command checks if the player have the capacity to hold the item (weight and inventory slot), here the item ID is 1201 and the amount is 2

Reply:
thanks @RagnaWay and @Atemo